js/index.js

```diff
@@ -0,0 +1,36 @@
+const addToInputList = document.querySelector("#addName");
+const addButton = document.querySelector("#addNameToList");
+const addToOutputList = document.querySelector("#outputList");
+
+// Function to append 
+function addToList() {
+    const ol = addToOutputList.querySelector("ol");
+    const li = document.createElement("li");
+    const paragraph = document.createElement("p");
+
+    li.innerHTML = addToInputList.value;
+
+    if (!li.innerHTML) {
+        console.warn('Input text field cannot be empty');
+    }
+    else {
+        ol.appendChild(li);
+        ol.appendChild(paragraph);
+        addToInputList.value = "";
+    }
+};
+
+
+// Execute when the user click on ADD button on Interface
+addButton.addEventListener("click", () => {
+        addToList();
+    }
+);
+
+// Execute when the user presses Enter key on the keyboard
+addToInputList.addEventListener("keypress", (event) => {
```

Comment:
I really like that you thought about this functionality for the user, allowing them to be able to submit the form while pressing the enter key makes your form accessible and we will talk about accessibility. 

However, I don't think the functionality is working as expected when I tested it.

Reply:
I was playing around with the functionality. It will be nice to include. While testing it on the chrome browser opened with the live server, the names were added after pressing the Enter key.

I noticed a deprecated notification in my JS code for the event.keyCode. I am not sure about the cause of the issue.

<img width="627" alt="image" src="https://user-images.githubusercontent.com/35523688/225396480-49fdca45-75a0-435b-b995-3eefd349a0ee.png">

Reply:
I replaced "event.keyCode === 13" with "event.key === 'Enter'". I am not sure if that will address the issue.

<img width="489" alt="image" src="https://user-images.githubusercontent.com/35523688/225398309-1802ecac-3580-435e-88fe-3b2bdb29f152.png">